deploy/helm/jumpstarter/charts/jumpstarter-controller/templates/cms/controller-cm.yaml

```diff
@@ -10,4 +10,5 @@ metadata:
     deployment.timestamp: {{ .Values.global.timestamp | quote }}
     {{ end }}
 data:
+  grpc: {{- .Values.grpcConfig | toYaml | indent 1 }}
```

Comment:
I believe we are over-complicating things by making a lot of small file config formats nested inside the configmap.

Reply:
Problem is configmap is of type `map[string][string]`, maybe make it accept a top level `config` field for everything?